### PR TITLE
HomeApp: add YouTube service webclip

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
Depends to https://github.com/Samsung/TAU/pull/1517 https://github.com/Samsung/TAU/pull/1519

[Issue] https://github.com/Samsung/TAU/issues/1518
[Problem] HomeApp: add YouTube webclip
[Solution]
     - new YouTube webclip has been added

[Screenshot]
obraz

### CODE REVIEW

**Review by singa2000 — APPROVED** [review]
It looks good

**Comment by singa2000** [comment]
@TomaszLukawskiSam 
Would you rebase the PR, please?

File: src/css/profile/mobile/common/card.less [hunk 1]
```diff
@@ -14,6 +14,33 @@
...
+				margin-right: 20 * @px_base;
+				margin-left: 10 * @px_base;
+			}
+		}
+		.ui-content-thumbnail {
+			width: 100%;
+		}
+		.ui-content {
+			.ui-title.ui-title-medium {
+				font-size: 18 * @px_base;
```
Comment (DohyungLim) [inline src/css/profile/mobile/common/card.less]: replace px_base to sp.

File: examples/mobile/HomeApp/js/app.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ import Storage from "./clipping-storage.js";
 			]
 		},
 		{
-			"appID": "vUf39tzQ4s.UIComponents",
+			"appID": "vUf39tzQ4s.Netflix",
```
Comment (DohyungLim) [inline examples/mobile/HomeApp/js/app.js]: are you changing the base to netflix?

**Review by DohyungLim — COMMENTED** [review]
2 comment

File: examples/mobile/HomeApp/js/app.js (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/js/app.js]: This app ID in this file has no real association with applications. It just has to be different.

File: src/css/profile/mobile/common/card.less (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/card.less]: Done.

**Pull request merged**